File: 2.0/java/campaigns/getCampaignSoundDataWav.java
import com.callfire.api.client.CallfireClient;

import java.io.InputStream;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("api_login", "api_password");
        InputStream is = callfireClient.campaignSoundsApi().getWav(12345L);
    }
}

File: 2.0/csharp/campaigns/getCampaignSoundDataWav.cs
using System.IO;
using CallfireApiClient;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        Stream stream = client.CampaignSoundsApi.GetWav(379506003);
    }
}
